### DISCUSSION TITLE
Can Blink Shell install Google SDK and use Google Compute SSH?

### DISCUSSION THREAD

**Discussion by tomanupong** [blinksh/blink#2078] [Q&A]
Can Blink Shell install Google SDK and use gcloud Compute SSH?

Thank you.

**Comment by carloscabanero** [comment]
We do not support the Cloud SDK as part of our Shell. But, you can install it as part of a Build machine and have it available in just a few seconds. That’s how we deploy our services to gcloud and Vercel!

**Reply by tomanupong** [reply]
@carloscabanero Thank you for the recommendation.
